R/auxiliary.R

```diff
@@ -202,3 +202,27 @@ process_row_details <- function(details_json) {
 
   tmp_result
 }
+
+#' Merge a list of regular expressions into a single one
+#'
+#' @param regex_l list of regular expressions to be merged.
+#'
+#' @returns Single regular expression string.
+#' @keywords internal
+merge_excluded_regex <- function(regex_l, remove_whitespace = TRUE) {
```

Comment:
`@param remove_whitespace` is not documented

Reply:
Added